src/Features/CSharp/Portable/ChangeSignature/CSharpChangeSignatureService.cs

```diff
@@ -143,6 +146,39 @@ private static int TryGetSelectedIndexFromDeclaration(int position, SyntaxNode m
             return parameters != null ? GetParameterIndex(parameters.Parameters, position) : 0;
         }
 
+        // Find the position to insert the new parameter.
+        // We will insert a new comma and a parameter.
+        private static int TryGetInsertPositionFromDeclaration(SyntaxNode matchingNode)
```

Comment:
how does this method ddiffer in VB?

Reply:
The difference is in types only: ParameterListSyntax, ParameterSyntax, SyntaxKind

---
In reply to: [367213745](https://github.com/dotnet/roslyn/pull/38748#discussion_r367213745) [](ancestors = 367213745)